pilot/pkg/kube/inject/inject.go

```diff
@@ -51,8 +51,9 @@ import (
 type annotationValidationFunc func(value string) error
 
 const (
-	annotationPolicy = "sidecar.istio.io/inject"
-	annotationStatus = "sidecar.istio.io/status"
+	annotationPolicy            = "sidecar.istio.io/inject"
+	annotationStatus            = "sidecar.istio.io/status"
+	annotationRewriteAppProbers = "sidecar.istio.io/rewriteAppProbers"
```

Comment:
Can you change to `rewriteAppHTTPProbers`? We use that naming throughout helm flags, and our code. Keeping it consistent can avoid confusion for future developers.

k8s app can also have tcp probers, which is not the under the scope of this feature.

Reply:
I have renamed to `rewriteAppHTTPProbers`. However, the one in template is `rewriteAppHTTPProbe`. Should we keep both in sync?